### ISSUE TITLE
Getting Unicode chars in the report while Newman running even if disableUnicode,noColor options are true.

### ISSUE THREAD

**Issue by SowmyaT** [postmanlabs/newman#1636]
1. Newman Version (can be found via `newman -v`): **3.8.3 or 3.10.0**
 2. OS details (type, version, and architecture): **OS X EL Capitan, 10.11.6**
 3. Are you using Newman as a library, or via the CLI? LIBRARY
 3. Did you encounter this recently, or has this bug always been there: **Recently after node upgraded**
 4. Expected behaviour: **Unicode chars should be removed**
 5. Command / script used to run Newman: **Script**
 6. Sample collection, and auxiliary files (minus the sensitive details):
 7. Screenshots (if applicable):
<img width="207" alt="screen shot 2018-07-24 at 11 51 40 pm" src="https://user-images.githubusercontent.com/17422722/43158161-a0c1da20-8f9c-11e8-9cdb-aa3e70cbca8a.png">

**Comment by codenirvana** [comment]
@SowmyaT Are you facing this issue in both `3.8.3` & `3.10.0`?

Also, can you share the following details, to help reproduce the bug:
- Options passed in `newman.run`
- Your Node version

**Comment by SowmyaT** [comment]
Yes @codenirvana , Am facing this issue in both 3.8.3 & 3.10.0.

- Options passed in `newman.run` only like,

newman.run({
	 reporters: ['cli', 'junit'],
	 collection: "Examples.postman_collection.json",
         disableUnicode: true,
         noColor: true
})

- Node version is v8.6.0 now

**Comment by codenirvana** [comment]
@SowmyaT I tried to reproduce the issue using the same options you provided.
`disableUnicode` is working fine for me, but I do see there's some problem with `noColor`.

I need to investigate further to pinpoint the issue, will update you once it is fixed.

**Comment by SowmyaT** [comment]
@codenirvana , We couldn't find this is the newman or node  or OS **version** issue. And we don't need to use `disableUnicode` & `noColor`. And also we updated the latest `node` versions as v8.11.3 and tried, still facing the same issue.

The same below library options worked fine earlier in that same Mac OS:
> newman.run({
> reporters: ['cli', 'junit'],
> collection: "Examples.postman_collection.json"
>});

<img width="187" alt="screen shot 2018-07-27 at 12 12 32 pm" src="https://user-images.githubusercontent.com/17422722/43305738-757823dc-9196-11e8-9581-903d8f46497f.png">

Output should be like the above without giving the disableUnicode & noColor as earlier. Please fix this asap.

**Comment by SowmyaT** [comment]
@codenirvana , What is the status of this issue.. Kindly let me know.

**Comment by codenirvana** [comment]
@SowmyaT a fix has been added in #1653 which will be rolled out with the `v4` release.

You can test this using [v4.0.0-beta.2](https://www.npmjs.com/package/newman/v/4.0.0-beta.2).
The color option is updated, you can check the updated API Reference [here](https://www.npmjs.com/package/newman/v/4.0.0-beta.2#newmanrunoptions-object--callback-function---run-eventemitter).

For your use case, try using:
```javascript
newman.run({
    reporters: ['cli', 'junit'],
    collection: 'collection.json',
    disableUnicode: true,
    color: 'off'
});
```

Let me know if the issue still persists.

**Comment by SowmyaT** [comment]
@codenirvana . Thanks for the Update! After installed the latest version `4.0.0-beta.2`.. Its works fine now.

**Comment by codenirvana** [comment]
@SowmyaT This has been fixed in Newman v4.0.0.
